tests/unit/ForeignJurisdiction.spec.ts

```diff
@@ -0,0 +1,104 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { mount } from '@vue/test-utils'
+import { ForeignJurisdiction } from '@/components/common'
+
+Vue.use(Vuetify)
+
+const vuetify = new Vuetify({})
+
+describe('ForeignJurisdiction', () => {
+  it('initializes correctly', () => {
+    const wrapper = mount(ForeignJurisdiction,
+      {
+        vuetify
+      })
+    const vm: any = wrapper.vm
+
+    expect(vm.selectedCountry).toBe('')
+    expect(vm.selectedRegion).toBe('')
+    expect(vm.countryNames.length).toBe(249)
+    expect(wrapper.emitted('valid')).toBeFalsy()
+    wrapper.destroy()
+  })
+
+  it('set correct country when selected via the country selector', async () => {
+    const wrapper = mount(ForeignJurisdiction,
+      {
+        vuetify
+      })
+    const vm: any = wrapper.vm
+
+    const countriesSelector = wrapper.find('#country-selector')
+    await countriesSelector.setValue('Lebanon')
+
+    expect(vm.selectedCountry).toBe('Lebanon')
```

Comment:
This should expect "LB" (abbreviation for Lebanon).

Reply:
Also you should check that an update event was emitted. (Ditto all unit tests where you can verify this.)

Reply:
Fix looks good:

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/357a5e6d-dd00-4c09-9f0f-d28f756640bb)

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/2772b9af-e6cc-4eb0-bd96-4ca1d1eb4f7d)

Reply:
Please check that country and region are correctly emitted.